lms/djangoapps/shoppingcart/processors/tests/test_CyberSource2.py

```diff
@@ -119,10 +119,30 @@ def test_get_signed_purchase_params(self):
         self.assertEqual(params['signature'], self._signature(params))
 
     # We patch the purchased callback because
+    # we're using the OrderItem base class, which throws an exception
+    # when item doest not have a course id associated
+    @patch.object(OrderItem, 'purchased_callback')
+    def test_process_payment_raises_exception(self, purchased_callback):  # pylint: disable=unused-argument
+        """
+        test that
```

Comment:
Please remove this docstring, so the test method name will show up in the console.